embedding-calculator/Dockerfile

```diff
@@ -47,6 +47,4 @@ RUN if [ -z $SKIP_TESTS  ]; then pytest -m "not performance" /app/ml/src; fi
 EXPOSE 3000
 
 COPY uwsgi.ini .
-ENV UWSGI_PROCESSES=${UWSGI_PROCESSES:-2}
```

Comment:
Let's leave it here. In case the user does not set these variables, it will still work